services-js/commissions-app/src/client/common/CommentInput.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export interface Props {
+  name: string;
+  placeholder: string;
+  title?: string;
+  error?: string;
+  value: string;
+  onChange: any;
+  onBlur: any;
+}
+
+export default class Comments extends React.Component<Props> {
+  render() {
+    return (
+      <div className="txt">
+        <textarea
+          rows={10}
+          title={this.props.title ? this.props.title : undefined}
+          name={this.props.name}
+          className={
+            this.props.error ? 'txt-f txt-f--sm txt-f--err' : 'txt-f txt-f--sm'
+          }
+          placeholder={this.props.placeholder}
+          value={this.props.value}
+          id="FeedbackForm-${this.props.name}"
+          onChange={this.props.onChange}
+          onBlur={this.props.onBlur}
+        />
+      </div>
```

Comment:
I just noticed you're not using `this.props.error` in this `render()` method.

Reply:
I'd expect you'd want the same error rendering as `TextInput` has.

Reply:
Kayla said she didn't want this required but if you want that error I can put it in